apis/apps/v1alpha1/cluster_types.go

```diff
@@ -384,6 +384,25 @@ type UserResourceRefs struct {
 	ConfigMapRefs []ConfigMapRef `json:"configMapRefs,omitempty"`
 }
 
+// InstanceUpdateStrategy indicates the strategy that the InstanceSet
+// controller will use to perform updates.
+type InstanceUpdateStrategy struct {
+	// Partition indicates the number of pods that should be updated during a rolling update.
+	// The remaining pods will remain untouched. This is helpful in defining how many pods
+	// should participate in the update process. The update process will follow the order
+	// of pod names in descending lexicographical (dictionary) order. The default value is
+	// Replicas (i.e., update all pods).
```

Comment:
Here, it should be clarified whether `Replicas` refers to componentSpec.replicas or instanceTemplate.replicas？